src/tests/functional/inference_engine/transformations/preprocessing_fusion_tests.cpp

```diff
@@ -672,3 +694,90 @@ TEST_F(TransformationTestsF, RICFusionSplitConcatDetectionNegative3) {
+        p.input(0).preprocess().convert_layout({2, 0, 1});
+        p.build();
+
+        manager.register_pass<ngraph::pass::TransposeSinking>();
+    }
+
+    {
+        auto input = std::make_shared<ngraph::opset6::Parameter>(ngraph::element::f16, ngraph::Shape{ 3, 64, 1 });
+        auto convert = std::make_shared<ngraph::opset6::Convert>(input, element::f32);
+        auto relu = std::make_shared<ngraph::opset6::Relu>(convert);
+
+        function_ref = std::make_shared<ngraph::Function>(ngraph::NodeVector{ relu }, ngraph::ParameterVector{ input });
+    }
+}
+
+TEST_F(TransformationTestsF, FuseScaleValue) {
+    {
+        auto input = create_param({ 1, 64, 64, 3 });
+        auto order = ngraph::opset6::Constant::create(ngraph::element::i64, ngraph::Shape{ 4 }, { 0, 3, 1, 2 });
+        auto transpose = std::make_shared<ngraph::opset6::Transpose>(input, order);
+        auto conv = create_conv(transpose, {6, 3, 3, 3});
+
+        function = std::make_shared<Function>(NodeVector{ conv }, ParameterVector{ input });
+
+        using namespace ov::preprocess;
+        PrePostProcessor p(function);
+        p.input(0).tensor().set_layout("NHWC");
+        p.input(0).preprocess().scale(1.3);
+        p.build();
+
+        manager.register_pass<pass::MOCTransformations>(false);
+    }
+
+    {
+        auto input = create_param({ 1, 64, 64, 3 });
+        auto order = ngraph::opset6::Constant::create(ngraph::element::i64, ngraph::Shape{ 4 }, { 0, 3, 1, 2 });
+        auto transpose = std::make_shared<ngraph::opset6::Transpose>(input, order);
+        auto conv = create_conv(transpose, {6, 3, 3, 3});
+        function_ref = std::make_shared<Function>(NodeVector{ conv }, ParameterVector{ input });
+    }
+
+    disable_rt_info_check();
+    enable_accuracy_check();
+}
+
+TEST_F(TransformationTestsF, FuseScaleValues) {
+    {
+        auto input = create_param({ 1, 64, 64, 3 });
+        auto order = ngraph::opset6::Constant::create(ngraph::element::i64, ngraph::Shape{ 4 }, { 0, 3, 1, 2 });
+        auto transpose = std::make_shared<ngraph::opset6::Transpose>(input, order);
+        auto conv = create_conv(transpose, {6, 3, 3, 3});
+
+        function = std::make_shared<Function>(NodeVector{ conv }, ParameterVector{ input });
+
+        using namespace ov::preprocess;
+        PrePostProcessor p(function);
+        p.input(0).tensor().set_layout("NHWC");
+        p.input(0).preprocess().scale({1.3, 0.2, 4.1});
+        p.build();
+
+        manager.register_pass<pass::MOCTransformations>(false);
+    }
+
+    {
+        auto input = create_param({ 1, 64, 64, 3 });
+        auto order = ngraph::opset6::Constant::create(ngraph::element::i64, ngraph::Shape{ 4 }, { 0, 3, 1, 2 });
+        auto transpose = std::make_shared<ngraph::opset6::Transpose>(input, order);
+        auto conv = create_conv(transpose, {6, 3, 3, 3});
+        function_ref = std::make_shared<Function>(NodeVector{ conv }, ParameterVector{ input });
```

Comment:
how accuracy is being checked? where did scale go?

Reply:
`enable_accuracy_check` enabled accuracy comparison between original and processed functions. See line below.